storage/txcache/score.go

```diff
@@ -38,25 +38,21 @@ func (computer *defaultScoreComputer) computeScore(scoreParams senderScoreParams
 
 // TODO (optimization): switch to integer operations (as opposed to float operations).
 func (computer *defaultScoreComputer) computeRawScore(params senderScoreParams) float64 {
-	allParamsDefined := params.fee > 0 && params.gas > 0 && params.size > 0 && params.count > 0
+	allParamsDefined := params.feeScore > 0 && params.gas > 0 && params.count > 0
 	if !allParamsDefined {
 		return 0
 	}
 
-	PPUMin := float64(computer.minGasPrice)
-	PPUAvg := float64(params.fee) / float64(params.gas)
-	PPUScore := math.Pow(PPUAvg/PPUMin, 3)
-
-	countPow2 := float64(params.count) * float64(params.count)
-	countScore := math.Log(countPow2+1) + 1
+	ppuMin := computer.txFeeHelper.minPricePerUnit()
+	ppuAvg := params.feeScore / (params.gas >> computer.txFeeHelper.gasLimitShift())
+	ppuRatio := ppuAvg << 3 / ppuMin
```

Comment:
Perhaps add the arithmetic expression equivalent above, as a comment (e.g. `// ppuRation := ppuAvg * 8 / ppuMin`) - here and below?

Reply:
added a comment for the shift
```
	// (<< 3)^3 and >> 9 cancel each other; used to preserve a bit more resolution
```